software/openvisualizer/openvisualizer/moteConnector/moteConnector.py

```diff
@@ -144,9 +144,73 @@ def _cmdToMote_handler(self,sender,signal,data):
                 self._sendToMoteProbe(
                     dataToSend = dataToSend,
                 )
+            elif data['action'][0]==moteState.moteState.SET_COMMAND:
+                # this is command for golden image
+                with self.stateLock:
+                    [success,dataToSend] = self._GDcommandToBytes(data['action'][1:])
+
+                if success == False:
+                    return
+
+                # print dataToSend
+                # send command to GD image
+                self._sendToMoteProbe(
+                    dataToSend = dataToSend,
+                )
             else:
                 raise SystemError('unexpected action={0}'.format(data['action']))
     
+    def _GDcommandToBytes(self,data):
+        
+        SUCCESS    = False
```

Comment:
when I read `SUCCESS`, I assume things have worked, which is not the case here.
Please rename to `outcome` (don't use all CAPS for variables)